clusterloader2/testing/experimental/storage/pod-startup/config.yaml

```diff
@@ -1,96 +1,151 @@
 # ASSUMPTIONS:
 # - Number of nodes should be divisible by NODES_PER_NAMESPACE (default 100).
 
-#Constants
-# TODO(mucahitkurt) copied from load tests, open for discussion.
-{{$NODE_MODE := DefaultParam .NODE_MODE "allnodes"}}
-{{$NODES_PER_NAMESPACE := DefaultParam .NODES_PER_NAMESPACE 100}}
-{{$PODS_PER_NODE := DefaultParam .PODS_PER_NODE 30}}
+# Cluster Variables
+  {{$NODE_MODE := DefaultParam .NODE_MODE "allnodes"}}
+  {{$NODES_PER_NAMESPACE := DefaultParam .NODES_PER_NAMESPACE 100}}
 
-{{$POD_TEMPLATE_PATH := DefaultParam .POD_TEMPLATE_PATH "../volume-types/emptydir/pod_with_emptydir.yaml"}}
-{{$LOAD_TEST_THROUGHPUT := DefaultParam .LOAD_TEST_THROUGHPUT 10}}
+# Framework Variables
+  {{$OBJ_CREATION_QPS := DefaultParam .OBJ_CREATION_QPS 10}}
+  {{$PARALLELISM_LIMITED := DefaultParam .PARALLELISM_LIMITED 1}}
+  {{$ENABLE_CHAOSMONKEY := DefaultParam .ENABLE_CHAOSMONKEY false}}
+  {{$TEST_NAME := DefaultParam .TEST_NAME "generic-test"}}
+  {{$GROUP := "pod-with-volumes-startup-latency"}}
+  {{$TUNING_SET := DefaultParam .TUNING_SET "UniformQPS"}}
 
-{{$VOLUMES_PER_POD := DefaultParam .VOLUMES_PER_POD 1}}
-{{$VOLUME_TEMPLATE_PATH := .VOLUME_TEMPLATE_PATH}}
-{{$PROVISION_VOLUME := DefaultParam .PROVISION_VOLUME false}}
+# Test Variales
+  {{$PODS_PER_NODE := .PODS_PER_NODE}}
+  {{$POD_TEMPLATE_PATH := .POD_TEMPLATE_PATH }}
+  {{$VOLUMES_PER_POD := .VOLUMES_PER_POD}}
+  {{$VOLUME_TEMPLATE_PATH := .VOLUME_TEMPLATE_PATH}}
+  {{$PROVISION_VOLUME := DefaultParam .PROVISION_VOLUME false}}
+  {{$VOL_SIZE := DefaultParam .VOL_SIZE "8Gi"}}
+  {{$WAIT_FOR_PVC := DefaultParam .WAIT_FOR_PVC false}}
```

Comment:
We don't need it - if there won't be PVC, the measurement will be no-op.
The less configuration options we have the easier to use it is.

Let's just run this measurement in every test - sometime it will just be no-op.

Reply:
It will still try to wait for totalVol PVCs right, which will time out and fail the test. How can I configure this to be a no-op?

Reply:
OK - I initially thought that the measurement simply looks into what PVCs exist and waits for all of them to be bounded. It's not exactly what we do, so in the current implementation I think we need to "if" that.
I would like to clean this up, but not necessary block this PR on it.